src/Mirza/BusinessRegistry/GenerateUtils.hs

```diff
@@ -0,0 +1,85 @@
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE RecordWildCards  #-}
+
+-- | Utility functions to generate test data using
+-- each service's client
+module Mirza.BusinessRegistry.GenerateUtils where
+
+import           Mirza.BusinessRegistry.Types          as BT
```

Comment:
Why are these grouped/sorted in this way? 

For consistency see how they are sorted here, https://github.com/data61/Mirza/pull/371/files#diff-c872a8a82faeca92b3b508b7222d3999L13

Reply:
Thanks for picking that up. I have removed the redundant imports in this module, and as such, the grouping became more consistent anyway :smile: